django/urls/conf.py

```diff
@@ -67,9 +69,22 @@ def _path(route, view, kwargs=None, name=None, Pattern=None):
             namespace=namespace,
         )
     elif callable(view):
+        # Fail early if the user forgot to call .as_view() on a CBV
+        if isinstance(view, type) and issubclass(view, View):
```

Comment:
Do we need the `if isinstance(view, type)`? If so, It would be good to add a test case for it.

Reply:
The `isinstance` is just there to guard `issubclass`, because `issubclass` throws if you give it something that isn't a type

```
>>> issubclass(int, object)
True
>>> issubclass(5, object)
Traceback (most recent call last):
  File "<pyshell#1>", line 1, in <module>
    issubclass(5, object)
TypeError: issubclass() arg 1 must be a class
```

Reply:
I've added a test for that